sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -25,12 +25,11 @@ and includes six main functions:
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-ai-textanalytics</artifactId>
-    <version>1.0.0-beta.5</version>
+    <version>1.0.0</version>
```

Comment:
we are good to do this ?

Reply:
https://github.com/Azure/azure-sdk-for-java/pull/11754#discussion_r435449260

Reply:
We are now https://github.com/Azure/azure-sdk/issues/1395#issuecomment-639143842 :)